test/verify/check-storage-mdraid

```diff
@@ -358,16 +361,17 @@ class TestStorageMdRaid(storagelib.StorageCase):
         self.wait_states({dev1: "In sync",
                           dev2: "In sync"})
 
-        # There should be a bitmap, and cockpit should not complain
+        # With mdadm < 4.4, there should be a bitmap, and cockpit should not complain.
 
-        self.assertIn("Consistency Policy : bitmap", m.execute("mdadm --misc -D /dev/md/md0"))
-        b.wait_not_present('.pf-v5-c-alert:contains("This MDRAID device has no write-intent bitmap")')
+        if mdadm_version < [4, 4]:
```

Comment:
This isn't quite right -- Debian has git snapshots, so the change happened between 4.3+20241108 and 4.3+20241202

Reply:
ah.

Reply:
So, what about just checking what mdadm has done and check whether Cockpit is consistent with that?

Reply:
Sounds good to me. Probably better than a `m.image in ["debian-testing", "arch"]` thing, as over time all the distros will get the new mdadm.